style.css

```diff
@@ -2,229 +2,780 @@
+  margin-bottom: 50px;
+}
+
+.heading {
   display: flex;
+  gap: 10px;
+  margin-left: 155px;
+  margin-top: 70px;
+  width: fit-content;
+  height: 100px;
+  align-items: center;
 }
 
-.intro-container {
-  background-color: #1c1a19;
+.title-1 {
+  font-size: 35px;
+  font-weight: none;
+  color: #091e42;
+  font-family: 'crete round', sans-serif;
+  width: 294px;
+  margin: 0px;
+  padding: 0px;
+}
+
+.line-top {
+  width: 705px;
+  opacity: 40%;
+  margin-left: 20px;
+}
+
+.work-grid-container {
+  display: flex;
+  margin-left: 155px;
+  margin-top: 150px;
+  gap: 20px;
+}
+
+.grid-item-1 {
+  width: 520px;
   height: auto;
-  width: 105.5vw;
+}
+
+.placer {
+  display: grid;
+  grid-template-columns: 390px;
+  row-gap: 10px;
+}
+
+.grid-item-2 {
+  font-family: 'Crete Round', sans-serif;
+  font-size: 32px;
+  color: #091e42;
+  margin-left: 15px;
+}
+
+.grid-item-3 {
+  font-family: 'Inter', sans-serif;
+  font-weight: 900;
+  font-size: 19px;
+  color: #091e42;
+  margin-left: 15px;
+}
+
+.grid-item-4 {
   display: flex;
-  position: static;
+  gap: 2vh;
+  padding-left: 0px;
+  margin-left: 15px;
 }
 
-header {
-  z-index: 1;
+.listing-section li {
+  width: fit-content;
 }
 
-.navigation {
-  height: 10.794602698650674vh;
-  min-width: 105.5vw;
-  max-width: 100%;
-  left: 0px;
-  top: 0px;
+.grid-item-4 li {
+  list-style-type: none;
+}
+
+.formating li {
+  margin-bottom: 10px;
+  padding: 10px;
+  text-align: center;
+}
+
+.grid-item-4 li:active {
+  background-color: white;
+  color: black;
+}
+
+.t-b-stack {
   border-radius: 0px;
-  background: #3e3c3c;
+  border: 2px solid #aaa;
+  padding: 10px 8px 10px 8px;
+  color: transparent;
 }
 
-.logo {
-  margin: 0vh 180.20989505247377vh 3.898050974512744vh 19.490254872563717vh;
+.t-b-stack:hover {
+  border: 2px solid #323264;
 }
 
-.logo-text {
-  font-family: var(--font-family), 'inter', sans-serif;
-  font-size: 20px;
-  font-weight: 700;
+.t-b-stack:active {
+  background-color: #091e42;
+  color: white;
+}
+
+.t-stack {
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  font-weight: 800;
   line-height: 20px;
   letter-spacing: 0em;
-  text-align: center;
+  text-align: left;
+}
+
+.grid-item-5 {
+  background-color: #c46710;
+  width: 150px;
+  height: 55px;
+  border: 0vh;
   color: white;
+  margin-left: 15px;
+}
+
+.grid-item-5:hover {
+  border: 2px solid #00008b;
+}
+
+.grid-item-5:active {
+  background-color: #7f3318;
+}
+
+.see-project-t {
+  font-family: 'Inter', sans-serif;
   align-items: center;
-  display: inline-flex;
-  position: absolute;
-  top: 3vh;
+  font-size: 20px;
+  color: white;
 }
 
-.item-nav {
+.all-sec-container {
   display: flex;
   flex-direction: row;
-  align-items: center;
-  padding: 0vh;
-  gap: 1.199400299850075vh;
-  font-family: var(--font-family), 'inter', sans-serif;
-  font-size: 15px;
-  font-weight: 600;
-  line-height: 20px;
+  flex-wrap: wrap;
+  row-gap: 30px;
+  margin-top: 70px;
+  margin-left: 145px;
+  width: 1145px;
```

Comment:
- [ ] I have noticed that you are using a fixed width on the section container. Due to this, the section container can not be responsive and therefore it is causing a horizontal scroll. The traditional horizontal scrollbar burdens the user by requiring constant attention and greater physical effort to maintain the dragging 😞. Kindly fix this as it will be the step to solve the horizontal scroll effect in your portfolio ☺️.